apps/emqx/src/emqx_limiter/src/emqx_limiter_client.erl

```diff
@@ -0,0 +1,344 @@
+%%--------------------------------------------------------------------
+%% Copyright (c) 2021 EMQ Technologies Co., Ltd. All Rights Reserved.
+%%
+%% Licensed under the Apache License, Version 2.0 (the "License");
+%% you may not use this file except in compliance with the License.
+%% You may obtain a copy of the License at
+%%
+%%     http://www.apache.org/licenses/LICENSE-2.0
+%%
+%% Unless required by applicable law or agreed to in writing, software
+%% distributed under the License is distributed on an "AS IS" BASIS,
+%% WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+%% See the License for the specific language governing permissions and
+%% limitations under the License.
+%%--------------------------------------------------------------------
+
+-module(emqx_limiter_client).
```

Comment:
Lots of places in this file use `Client` as Var name but it is actually `limiter()`, `ref_limiter()` or `undefined`.

Using var `Client` in this file without considering the context makes it a lot difficult to read. Maybe it is my personal preference. 

Pls consider renaming them.